linux_os/guide/system/software/updating/ensure_gpgcheck_never_disabled/ansible/shared.yml

```diff
@@ -1,20 +1,20 @@
 # platform = multi_platform_rhel,multi_platform_ol,multi_platform_fedora,multi_platform_rhv
 # reboot = false
-# strategy = unknown
+# strategy = enable
 # complexity = low
 # disruption = medium
-#
-- name: Find All {{{ pkg_manager }}} Repositories
-  find:
-    paths: "/etc/yum.repos.d/"
-    patterns: "*.repo"
-    contains: ^\[.+]$
-  register: yum_find
+- name: Grep for {{{ pkg_manager }}} repo section names
+  shell: grep -HEr '^\[.+\]' -r ./etc/yum.repos.d/
```

Comment:
the dot before /etc shouldn't be there

Reply:
Oops, thanks for the catch @jan-cerny - left over from testing. Just pushed a fix.